### PR TITLE
[WIP] add move function

### PR DESCRIPTION
**Pull request by adbenitez**
partially tackles #351 adding a `window.set_window_position(x, y)` implemented **only for gtk and qt** platforms, sorry, I am not a Mac or Windows and have no experience with `cocoa` or `winforms`.
Leaving here as base for someone else to implement the rest of platforms :)

### CODE REVIEW

**Comment by adbenitez** [comment]
oops! sorry :( I for got I have autopep8 to format automatically on save in my Emacs, feel free to modify this PR if you don't want to change the formatting

**Comment by r0x0r** [comment]
Thanks for tackling this! Could you resubmit the PR with autoformat off? 
Another thing, a more apt name for this function could be `window.move(x,y)`

**Comment by adbenitez** [comment]
I will go offline for today, I will not have internet until tomorrow, will do it then, sorry :(

**Comment by adbenitez** [comment]
@r0x0r 
I blindly implemented the move function in winforms.py, it *should* work ;)
I noticed here that the code is using a int for the flags of `setWindowPos` I think for better documentation would be better to define some global constants with the flags and use them as I did in the move function, also the `set_window_size` function may use other flags like `SWP_NOMOVE` to just pass the width and height and ignore the x and y position since this is not a move operation, so you don't need to use `self.Location.X` etc.
(I don't know anything about this windll thing I am just guessing so sorry if I am making a dummy suggestion ;)

**Comment by adbenitez** [comment]
about `qt.py`  tell me if you are ok with the internal names `window_move_trigger` and `on_window_move`?

**Comment by adbenitez** [comment]
added the implementation for `cocoa.py` this really needs a review I am not really sure of what I have done there, seems like y coordinates are flipped in cocoa

**Comment by acbetter** [comment]
> added the implementation for `cocoa.py` this really needs a review I am not really sure of what I have done there, seems like y coordinates are flipped in cocoa

`cocoa.py` works well with this test file on my macOS.

```python
# test_move_window.py

import time
import webview
from .util import run_test


def test_move_window():
    window = webview.create_window("Set Window Size Test", "https://www.example.org")
    run_test(webview, window, move_window)


def move_window(window):
    window.move(50, 500)
    time.sleep(1)
    window.move(500, 50)
    time.sleep(1)
    window.move(500, 500)
    time.sleep(1)
    window.move(50, 50)
    time.sleep(1)
```

File: webview/platforms/gtk.py [hunk 1]
```diff
@@ -400,6 +403,12 @@ def _set_window_size():
     glib.idle_add(_set_window_size)
 
 
+def move(x, y, uid):
+    def _set_window_position():
+        BrowserView.instances[uid].set_window_position(x, y)
```
Comment (r0x0r) [inline webview/platforms/gtk.py]: set_window_position instead of move

**Comment by r0x0r** [comment]
I have made some fixes and moved the PR to `move-window` branch. Some things left to do:
- Window centering on GTK and Windows.
- Cocoa screen coordinate calculation in a multiscreen setup.

**Pull request merged**